src/static/helpers/slasClient.ts

```diff
@@ -49,14 +49,15 @@ export interface ISlasClient {
     options: {
       parameters?: {
         organizationId?: string;
-      };
+      } & { [key in `c_${string}`]: any };
```

Comment:
Can we use unknown over any type for this custom key value? 🤔 
```suggestion
      } & { [key in `c_${string}`]: unknown };
```

Reply:
Ah good callout, `unknown` is the the better type to use in this situation but unfortunately using unknown causes typescript errors :( 

```
Type 'Partial<{ [x: `c_${string}`]: unknown; }>' is not assignable to type 'QueryParameters'.
  '`c_${string}`' and 'string' index signatures are incompatible.
    Type 'unknown' is not assignable to type 'string | number | boolean | string[] | number[]'.
```

It's cause we send the query parameters to `commerce-sdk-core` which expects query parameters as typed:
`string | number | boolean | string[] | number[]`. So we can do:

```suggestion
      } & { [key in `c_${string}`]: string | number | boolean | string[] | number[] };
```

The only concern I have with this approach is that customers could theoretically pass other types to custom query parameters like `boolean []`. For that reason, I'm leaning towards using `any`, but I could be convinced otherwise